monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/pages/CarbonApplications.js

```diff
@@ -41,15 +40,7 @@ export default function CarbonApplications() {
     const selectedNodeList = useSelector(state => state.nodeList);
 
     React.useEffect(() => {
-        var nodeListQueryParams="";
-        selectedNodeList.filter(node => {
```

Comment:
I have the same concern as https://github.com/wso2/product-mi-tooling/pull/208/files#r676335124